packages/solana/src/client.ts

```diff
@@ -140,7 +140,19 @@ export class Web3Modal extends Web3ModalScaffold {
         })
 
         return signature as string
-      }
+      },
+
+      sendTransaction: async () => {
```

Comment:
These are just to handle type errors. It's not available in Solana right now and won't be used. Maybe we could handle these better?

Reply:
I would at least leave a comment for future reviewers. 
Technically this shouldn't even be here, what type error is requiring it?

Reply:
If I remove them here, I need to make the `sendTransaction`, `parseUnits` and `formatUnits` methods as optional in the ConnectionController.

This would force us to make a type check everytime when we use these methods, like `ConnectionController?.sendTransaction(..)`, or even we need to me the check on the top level of the function we are using them:

```js
function createConvertTransaction() {
 const availableToSendTransaction = !!ConnectionController?.sencTransaction
...
}
````

etc. It depends on us that we could both make them optional methods or use empty functions on the Solana client to not raise optional property access things.

It's matter the answer of: 
- "Will we have the same features in the Solana?"
- "If yes, is the Solana methods working the same way EVM works?" (I guess the answer is no)
- "If not, is it worth make the other client's work harder? etc."

We should double check these with the Boosty tea, but I do believe it's too early to consider wallet features in Solana network for now.

Reply:
I think this is because the client requires this right?

Reply:
Totally. ConnectorController is the place we are using the same methods higher level on the UI side or in other controllers. So we are making the API same for all clients and use them in same place.